src/App.jsx

```diff
@@ -1,7 +1,31 @@
 import ReactDOM from "react-dom/client";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import AddPage from "./pages/AddPage/AddPage";
+import AllCompletedUncompleted from "./pages/AllCompletedUncompleted/AllCompletedUncompleted";
+import ConfigurationPage from "./pages/ConfigurationPage/ConfigurationPage";
+import Header from "./components/Header/Header";
+import Navbar from "./components/Navbar/Navbar";
 
 const App = () => {
-
+  return (
+    <BrowserRouter>
+      <>
+        <div className="site-wrapper">
+          <div className="container">
+            <main className="content">
+              <Header />
+              <Navbar />
+              <Routes>
+                <Route path="/add" element={<AddPage />} />
+                <Route path="/" element={<AllCompletedUncompleted />} />
```

Comment:
You should not delete all the routes, Now `uncompleted` and `completed` links are broken. Bring them back and render the same component. (`TodoList`).

The idea of having just one component comes from the fact that they all have the same ui and rendering logic. So, all of them should render the same component but should have different routes. Later on, we get to that step that we pass extra parameter to url and get that inside component and apply the filtering to the list.

Reply:
Done.